network/data_link/wifi.c

```diff
@@ -52,7 +52,7 @@
 /********************** Macros and Constants Definitions **********************/
 /******************************************************************************/
 
-#define CONNECTION_BUFFER_SIZE 1500
```

Comment:
typo in commit message: increas -> increase

Reply:
done